decidim-proposals/app/mailers/decidim/proposals/admin/proposals_valuator_mailer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      class ProposalsValuatorMailer < Decidim::ApplicationMailer
+        include Decidim::TranslationsHelper
+        include Decidim::SanitizeHelper
+        include Decidim::ApplicationHelper
+
+        def notify_proposals_valuator(user, admin, proposals, proposal)
+          @valuator_user = user
+          @admin = admin
+          @proposals = proposals
+          @proposal = proposal
+          @organization = user.organization
+
+          mail to: "#{user.name} <#{user.email}>",
+               from: "#{admin.name}, #{admin.email}",
+               subject: t(".subject")
```

Comment:
```suggestion
        with_user(user) do
          mail to: "#{user.name} <#{user.email}>",
               from: "#{admin.name}, #{admin.email}",
               subject: t(".subject")
        end
```

Using the `with_user` helper will send the email in the language this user has Decidim configured. It's a little improvement